src/client/Client.cc

```diff
@@ -152,6 +152,17 @@ bool Client::CommandHook::call(std::string_view command,
     m_client->dump_mds_sessions(f.get());
   else if (command == "dump_cache")
     m_client->dump_cache(f.get());
+  else if(command == "dump_inode"){
+        std::string str_inode_no;
+        cmd_getval(m_client->cct, cmdmap, "inode_no", str_inode_no);
+        uint64_t inode_no = strtoull(str_inode_no.c_str(), NULL, 0);
+        if ((inode_no == 0) && f) {
+            f->open_array_section("cache");
+            f->close_section();
+        } else {
+            m_client->dump_cache(f, inode_no);
```

Comment:
`m_client->dump_cache(f.get(), inode_no);` when using `Formatter *f` in dump_cache parameter.